Gems/Multiplayer/Code/Source/NetworkEntity/NetworkEntityManager.cpp

```diff
@@ -467,6 +467,54 @@ namespace Multiplayer
         return netEntityId;
     }
 
+    AzFramework::EntitySpawnTicket NetworkEntityManager::RequestNetSpawnableInstantiation(const AZ::Data::Asset<AzFramework::Spawnable>& rootSpawnable)
+    {
+        const AzFramework::Spawnable::EntityList& entityList = rootSpawnable->GetEntities();
+        if (entityList.size() == 0)
+        {
+            AZ_Assert(false, "RequestNetSpawnableInstantiation: No entities in the spawnable %s", rootSpawnable.GetHint().c_str())
+            return {};
```

Comment:
An assert seems harsh here since it doesn't look like a programmer's error what asset content is passed here. AZ_Error seems to a better fit.

Reply:
True, changed to Error, thanks!